core and subdirs root permissions not changed to user

With version https://github.com/OCR-D/ocrd_all/commit/a709c32ac6d820641e926fae36a3c910e23c192a and ``sudo make deps-ubuntu``, ``core`` and its subdirs are created with root permissions but not changed to user. 

```
~/tmp/dev/ocrd/ocrd_all/core  ➦ dc3ffe4d  ls -la 
total 160
drwxr-xr-x 12 cnd  cnd   4096 Aug 24 13:13 .
drwxr-xr-x 33 cnd  cnd   4096 Aug 24 13:08 ..
-rw-r--r--  1 root root 31410 Aug 24 13:02 CHANGELOG.md
drwxr-xr-x  2 root root  4096 Aug 24 13:02 .circleci
-rw-r--r--  1 root root   449 Aug 24 13:02 .coveragerc
drwxr-xr-x  2 root root  4096 Aug 24 13:02 dev
-rw-r--r--  1 root root   766 Aug 24 13:02 Dockerfile
-rw-r--r--  1 root root    39 Aug 24 13:02 .dockerignore
drwxr-xr-x  3 root root  4096 Aug 24 13:02 docs
-rw-r--r--  1 root root    31 Aug 24 13:02 .envrc
-rw-r--r--  1 root root    29 Aug 24 13:02 .git
-rw-r--r--  1 root root  1445 Aug 24 13:02 .gitignore
-rw-r--r--  1 root root   166 Aug 24 13:02 .lgtm.yml
-rw-r--r--  1 root root 11357 Aug 24 13:02 LICENSE
-rw-r--r--  1 root root  5901 Aug 24 13:02 Makefile
drwxr-xr-x  4 root root  4096 Aug 24 13:02 ocrd
drwxr-xr-x  3 root root  4096 Aug 24 13:02 ocrd_modelfactory
drwxr-xr-x  4 root root  4096 Aug 24 13:02 ocrd_models
drwxr-xr-x  3 root root  4096 Aug 24 13:02 ocrd_utils
drwxr-xr-x  3 root root  4096 Aug 24 13:02 ocrd_validators
-rw-r--r--  1 root root  1392 Aug 24 13:02 .pylintrc
-rw-r--r--  1 root root  4120 Aug 24 13:02 README.md
drwxr-xr-x  3 cnd  cnd   4096 Aug 24 13:13 repo
-rw-r--r--  1 root root   182 Aug 24 13:02 requirements_test.txt
-rw-r--r--  1 root root   452 Aug 24 13:02 .scrutinizer.yml
drwxr-xr-x  7 root root  4096 Aug 24 13:02 tests
-rw-r--r--  1 root root   294 Aug 24 13:02 tox.ini
-rw-r--r--  1 root root   289 Aug 24 13:02 .travis.yml
-rw-r--r--  1 root root   112 Aug 24 13:02 .vimrc
```

This breaks installation with ``make all`` due to permission error:

```bash
make[1]: Entering directory '/home/cnd/tmp/dev/ocrd/ocrd_all/core'
mkdir -p repo/
git clone https://github.com/OCR-D/spec "repo/spec"
Cloning into 'repo/spec'...
remote: Enumerating objects: 169, done.
remote: Counting objects: 100% (169/169), done.
remote: Compressing objects: 100% (117/117), done.
remote: Total 1596 (delta 103), reused 100 (delta 52), pack-reused 1427
Receiving objects: 100% (1596/1596), 16.00 MiB | 4.91 MiB/s, done.
Resolving deltas: 100% (975/975), done.
cp repo/spec/ocrd_tool.schema.yml ocrd_validators/ocrd_validators/ocrd_tool.schema.yml
cp: cannot create regular file 'ocrd_validators/ocrd_validators/ocrd_tool.schema.yml': Permission denied
Makefile:121: recipe for target 'spec' failed
make[1]: *** [spec] Error 1
make[1]: Leaving directory '/home/cnd/tmp/dev/ocrd/ocrd_all/core'
Makefile:158: recipe for target '/home/cnd/tmp/dev/ocrd/ocrd_all/venv/bin/ocrd' failed
make: *** [/home/cnd/tmp/dev/ocrd/ocrd_all/venv/bin/ocrd] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -709,7 +709,7 @@ clean-tesseract:
 deps-ubuntu:
 	apt-get -y install git parallel
 	$(MAKE) deps-ubuntu-modules
-	chown -R --reference=$(CURDIR) .git $^
+	chown -R --reference=$(CURDIR) .git $(OCRD_MODULES)
 # prevent the sem commands during above module updates from imposing sudo perms on HOME:
 	chown -R --reference=$(HOME) $(HOME)/.parallel
 

